specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json

```diff
@@ -1857,6 +2366,302 @@
+    "SqlContainerCreateUpdateParameters": {
+      "description": "Parameters to create and update Cosmos DB SQL container.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties to create and update Azure Cosmos DB SQL container.",
+          "$ref": "#/definitions/SqlContainerCreateUpdateProperties"
+        }
+      },
+      "required": [
+        "properties"
+      ]
+    },
+    "SqlContainerCreateUpdateProperties": {
+      "description": "Properties to create and update Azure Cosmos DB SQL container.",
+      "type": "object",
+      "properties": {
+        "resource": {
+          "description": "The standard JSON format of a SQL container",
+          "$ref": "#/definitions/SqlContainerResource"
+        },
+        "options": {
+          "description": "A key-value pair of options to be applied for the request. This corresponds to the headers sent with the request.",
+          "$ref": "#/definitions/CreateUpdateOptions"
+        }
+      },
+      "required": [
+        "resource",
+        "options"
+      ]
+    },
+    "SqlDatabaseResource": {
+      "type": "object",
+      "description": "Cosmos DB SQL database id object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Name of the Cosmos DB SQL database"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "SqlContainerResource": {
+      "type": "object",
+      "description": "Cosmos DB SQL container resource object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Name of the Cosmos DB SQL container"
+        },
+        "indexingPolicy": {
+          "$ref": "#/definitions/IndexingPolicy",
+          "description": "The configuration of the indexing policy. By default, the indexing is automatic for all document paths within the SQL container"
+        },
+        "partitionKey": {
+          "$ref": "#/definitions/PartitionKey",
+          "description": "The configuration of the partition key to be used for partitioning data into multiple partitions"
+        },
+        "defaultTtl": {
+          "type": "integer",
+          "description": "Default time to live"
+        },
+        "uniqueKeyPolicy": {
+          "$ref": "#/definitions/UniqueKeyPolicy",
+          "description": "The unique key policy configuration for specifying uniqueness constraints on documents in the collection in the Azure Cosmos DB service."
+        },
+        "conflictResolutionPolicy": {
+          "$ref": "#/definitions/ConflictResolutionPolicy",
+          "description": "The conflict resolution policy for the SQL container."
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "IndexingPolicy": {
+      "type": "object",
+      "description": "Cosmos DB indexing policy",
+      "properties": {
+        "automatic": {
+          "type": "boolean",
```

Comment:
Generally speaking, string enums (with values of true, false) are preferred to booleans as they can be extended later without being a breaking change.